linode/childaccount/datasource_test.go

```diff
@@ -10,6 +10,21 @@ import (
 	"github.com/linode/terraform-provider-linode/v2/linode/childaccount/tmpl"
 )
 
+func TestSmokeTests_childaccount(t *testing.T) {
+	tests := []struct {
+		name string
+		test func(*testing.T)
+	}{
+		{"TestAccDataSourceChildAccount_basic_smoke", TestAccDataSourceChildAccount_basic_smoke},
+	}
+
+	for _, tt := range tests {
+		if acceptance.Contains(acceptance.SmokeTests["childaccount_test"], tt.name) {
+			t.Run(tt.name, tt.test)
+		}
+	}
```

Comment:
So can't we just include all smoke tests in the `tests` variable here without the global `SmokeTests`?

Reply:
That's correct. I had to do this because `func TestSmokeTests_...` need to be defined in every test file that contains smoke test(s) and I wanted one global place where we can see the entire suite easily.

Reply:
That is true, I will remove the global map for now then